src/components/NurseryWithdrawals/NurseryWithdrawals.tsx

```diff
@@ -1,10 +1,119 @@
+import { Box, Grid, Theme, Typography, useTheme } from '@mui/material';
+import TextField from '@terraware/web-components/components/Textfield/Textfield';
+import { useEffect, useRef, useState } from 'react';
+import strings from 'src/strings';
 import { ServerOrganization } from 'src/types/Organization';
+import PageHeaderWrapper from '../common/PageHeaderWrapper';
 import TfMain from '../common/TfMain';
+import { makeStyles } from '@mui/styles';
+import { Table, TableColumnType } from '@terraware/web-components';
+import { listNurseryWithdrawals } from 'src/api/tracking/withdrawals';
+import { SearchResponseElement } from 'src/api/search';
+import WithdrawalLogRenderer from './WithdrawalLogRenderer';
+import { APP_PATHS } from 'src/constants';
+import { useHistory } from 'react-router-dom';
 
 type NurseryWithdrawalsProps = {
   organization: ServerOrganization;
 };
 
+const useStyles = makeStyles((theme: Theme) => ({
+  searchField: {
+    width: '300px',
+  },
+}));
+
+const columns: TableColumnType[] = [
+  { key: 'withdrawnDate', name: strings.DATE, type: 'string' },
+  { key: 'purpose', name: strings.PURPOSE, type: 'string' },
+  { key: 'facility_name', name: strings.FROM_NURSERY, type: 'string' },
+  { key: 'destinationName', name: strings.DESTINATION, type: 'string' },
+  { key: 'plotNames', name: strings.TO_PLOT, type: 'string' },
+  { key: 'speciesScientificNames', name: strings.SPECIES, type: 'string' },
+  { key: 'totalWithdrawn', name: strings.TOTAL_QUANTITY, type: 'string' },
+  { key: 'hasReassignments', name: '', type: 'string' },
```

Comment:
would this be better named simply `'reassignments'`, as `'hasReassignments'` implies a boolean value?

Reply:
The "keys" correspond to the values that come from the search api and the "name" is the title of the column in the table

Reply:
gotcha 👍 Thanks!